pkg/util/helper/policy.go

```diff
@@ -118,6 +118,15 @@ func IsReplicaDynamicDivided(strategy *policyv1alpha1.ReplicaSchedulingStrategy)
 	}
 }
 
+// IsReplicaDuplicated checks if a PropagationPolicy schedules replicas as duplicated
+func IsReplicaDuplicated(strategy *policyv1alpha1.ReplicaSchedulingStrategy) bool {
+	if strategy == nil || strategy.ReplicaSchedulingType == policyv1alpha1.ReplicaSchedulingTypeDuplicated {
+		return true
+	}
+
+	return false
+}
```

Comment:
I guess we can have a helper function at `pkg/apis/policy/v1alpha1/propagation_helper.go` like:
```golang
func (p *Placement) ReplicaSchedulingType() ReplicaSchedulingType {
	if p.ReplicaScheduling == nil {
		return ReplicaSchedulingTypeDuplicated
	}

	return p.ReplicaScheduling.ReplicaSchedulingType
}
```

Reply:
I'll open an issue for this.

Reply:
Done. https://github.com/karmada-io/karmada/issues/3347